modules/mono/mono_gd/gd_mono.cpp

```diff
@@ -864,8 +864,8 @@ Error GDMono::reload_scripts_domain() {
 				metadata_set_api_assembly_invalidated(APIAssembly::API_EDITOR, true);
 			}
 
-			Error err = _unload_scripts_domain();
-			if (err != OK) {
+			Error err2 = _unload_scripts_domain();
+			if (err2 != OK) {
```

Comment:
It would be better to just re-use `err` here instead of declaring a new local.

Reply:
Works for me.